README.md

```diff
@@ -17,4 +17,6 @@ Create a file in the project directory named **packages.yaml.inc** and include y
 - geteltorito
 ```
 
+Optionally, use the included script `aur-packages-from-user.py` to generate it automatically.
```

Comment:
Thanks for the PR!

This seems to introduce a use case I hadn't thought of, in that perhaps a package maintainer wants to build all their own packages in Jenkins.  For the original use case, it was just to bring in a list of packages from multiple maintainers so that you could host your own binary Arch repo for personal use.  

Anyways, I would like to see this reflected in the README, something like "if you'd like to build all the packages from a particular AUR maintainer/username, you may use the included script...".